service-account/bound-sa-token.md

```diff
@@ -1,122 +0,0 @@
-%title: Bound Service Account Token
-%author: Krzysztof Ostrowski
-%date: 10.03.2023
-
--> # Bound Service Account Token <-
-
---------------------------------------------------
--> # Table of Contents <-
-==============
-
--> # What is a Service Account Token  <-
--> # Service Account Token as a YAML <-
--> # Old Service Account Token Behavior  <-
--> # What is a Bound Service Account Token  <-
--> # Lord of Tokens: Kubelet <-
--> # Lord of ServiceAccounts: Service Account Admission Controller <-
--> # Bound Service Account Token in use by a Pod <-
-
--------------------------------------------------
--> # What is a Service Account Token  <-
-
-An **identity** that an instance or an application can use to make **API requests**:
-
-1. Intended to be used for **processes**.
-2. Bound to a **namespace**.
-3. Being associated with a ServiceAccount provides a **Bearer Token**.
```

Comment:
I am not sure I understand, this should be rephrased.

Reply:
So if a pod is associated with a Service Account, you will be provided a Bearer Token, the Service ACcount Token...